spec/features/etd_creation_spec.rb

```diff
@@ -207,12 +207,13 @@
     Timeout.timeout(Settings.timeouts.workflow) do
       loop do
         visit "#{Settings.argo_url}/view/#{prefixed_druid}"
-        break if page.has_text?("Embargoed until #{embargo_date.to_formatted_s(:long)}", wait: 1)
+        break if page.has_text?("Embargoed until #{embargo_date.to_formatted_s(:long)}",
+                                wait: Settings.timeouts.etd.embargo_poll_interval)
       end
     end
     expect(page).to have_text(dissertation_title)
     apo_element = find_table_cell_following(header_text: 'Admin policy')
-    expect(apo_element.first('a')[:href]).to have_text('druid:bx911tp9024') # this is hardcoded in hydra_etd app
+    expect(apo_element.first('a')[:href]).to include('druid:bx911tp9024') # this is hardcoded in hydra_etd app
```

Comment:
i'm not sure how this passed for other folks before?  when the tests got here for me with the old code, it complained that `apo_element.first('a')[:href]` was a `String`, and so the `has_text?` method was unavailable.  e.g.
```ruby
expected "https://argo-stage.stanford.edu/view/druid:bx911tp9024" to respond to `has_text?`
```

Reply:
I wound up having to change this in my PR too.